src/openrct2/ride/coaster/BolligerMabillardTrack.cpp

```diff
@@ -27,45 +27,21 @@ void bolliger_mabillard_track_flat(
 {
     if (tileElement->AsTrack()->HasChain())
     {
-        switch (direction)
-        {
-            case 0:
-                sub_98197C_rotated(
-                    session, direction, session->TrackColours[SCHEME_TRACK] | 17486, 0, 0, 32, 20, 3, height, 0, 6, height);
-                break;
-            case 1:
-                sub_98197C_rotated(
-                    session, direction, session->TrackColours[SCHEME_TRACK] | 17487, 0, 0, 32, 20, 3, height, 0, 6, height);
-                break;
-            case 2:
-                sub_98197C_rotated(
-                    session, direction, session->TrackColours[SCHEME_TRACK] | 17488, 0, 0, 32, 20, 3, height, 0, 6, height);
-                break;
-            case 3:
-                sub_98197C_rotated(
-                    session, direction, session->TrackColours[SCHEME_TRACK] | 17489, 0, 0, 32, 20, 3, height, 0, 6, height);
-                break;
-        }
+        uint32_t imageIds[] = { session->TrackColours[SCHEME_TRACK] | 17486, session->TrackColours[SCHEME_TRACK] | 17487,
```

Comment:
Take out `session->TrackColours[SCHEME_TRACK] | ` and move that to the `sub_98197C_rotated()` call.

Reply:
Oops this is called in a lot of other places...